build/MobileCoreServices/lib/MobileCoreServicesLib.vcxproj

```diff
@@ -19,8 +19,12 @@
     </ProjectConfiguration>
   </ItemGroup>
   <ItemGroup>
-    <ClangCompile Include="..\..\..\Frameworks\MobileCoreServices\UTType.mm" />
-    <ClangCompile Include="..\..\..\Frameworks\MobileCoreServices\UTTypeInternalMap.mm" />
+    <ClangCompile Include="..\..\..\Frameworks\MobileCoreServices\UTType.mm">
+      <CompileAs Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">CompileAsCpp</CompileAs>
```

Comment:
Right now, these files are only being compiled as C++ for Debug+Win32!

Two suggestions:
1. Make this apply to all configurations.
2. Rename those files to .cpp to clear up the naming confusion.